modulemd/v2/tests/test-modulemd-modulestream.c

```diff
@@ -672,6 +720,12 @@ main (int argc, char *argv[])
               NULL,
               module_stream_v2_test_depends_on_stream,
               NULL);
+  g_test_add ("/modulemd/v2/modulestream/community",
+              ModuleStreamFixture,
+              NULL,
+              NULL,
+              module_stream_test_arch,
```

Comment:
```suggestion
              module_stream_test_community,
```

Your test isn't actually being run if you don't have the correct function name here.